<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>colorWidget</class>
 <widget class="QWidget" name="colorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>329</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>FontAwesome</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <layout class="QFormLayout" name="formLayout">
         <property name="verticalSpacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="rgbLabel">
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>FreeSans</family>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: grey</string>
           </property>
           <property name="text">
            <string>RGB</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="margin">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QLineEdit" name="rgbValue">
              <property name="font">
               <font>
                <family>FontAwesome</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">padding: 2px</string>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QWidget" name="widget_5" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QLineEdit" name="htmlValue">
              <property name="font">
               <font>
                <family>FontAwesome</family>
                <pointsize>9</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">padding: 2px</string>
              </property>
              <property name="readOnly">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="htmlLabel">
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>FreeSans</family>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: grey</string>
           </property>
           <property name="text">
            <string>HTML</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="margin">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="alphaLabel">
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>FreeSans</family>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: grey</string>
           </property>
           <property name="text">
            <string>Alpha</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="margin">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QWidget" name="alphaContainer" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QSlider" name="alphaValue">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="alphaValueTxt">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>FontAwesome</family>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>0.0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="margin">
               <number>4</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="scaleLabel">
           <property name="minimumSize">
            <size>
             <width>64</width>
             <height>24</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>64</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>FreeSans</family>
             <pointsize>8</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: grey</string>
           </property>
           <property name="text">
            <string>Scale</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="margin">
            <number>4</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QWidget" name="widget_3" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QSlider" name="scaleValue">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="scaleValueTxt">
              <property name="minimumSize">
               <size>
                <width>64</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>FontAwesome</family>
                <pointsize>8</pointsize>
               </font>
              </property>
              <property name="text">
               <string>1.0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
              <property name="margin">
               <number>4</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>alphaValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>colorWidget</receiver>
   <slot>alphaChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>407</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>scaleValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>colorWidget</receiver>
   <slot>scaleChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>175</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>htmlValue</sender>
   <signal>editingFinished()</signal>
   <receiver>colorWidget</receiver>
   <slot>htmlChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>334</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rgbValue</sender>
   <signal>editingFinished()</signal>
   <receiver>colorWidget</receiver>
   <slot>rgbChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>105</x>
     <y>371</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>activateValue(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
  <slot>valueUpdated()</slot>
  <slot>resetValue()</slot>
  <slot>maxUpdated()</slot>
  <slot>pickColor()</slot>
  <slot>alphaChanged()</slot>
  <slot>htmlChanged()</slot>
  <slot>scaleChanged()</slot>
  <slot>rgbChanged()</slot>
 </slots>
</ui>
